subseas_flex_fcst/maproom.py

```diff
@@ -39,6 +46,53 @@
 
 APP.layout = layout.app_layout
 
+
+def adm_borders(shapes):
+    with psycopg2.connect(**CONFIG["db"]) as conn:
+        s = sql.Composed(
+            [
+                sql.SQL("with g as ("),
+                sql.SQL(shapes),
+                sql.SQL(
+                    """
+                    )
+                    select
+                        g.label, g.key, g.the_geom
```

Comment:
If you remove the key from the SQL query in the config file, also remove `g.key` here.